File: tests/golang/testB.go
package main

import "fmt"

func printHelloTestB1() {
    fmt.Println("Hello, World!")
    for i := 0; i < 5; i++ {
        fmt.Printf("This is line %d\n", i)
        if i%2 == 0 {
            fmt.Println("Even number")
        } else {
            fmt.Println("Odd number")
        }
    }
}

func printHelloTestB2() {
    fmt.Println("Hello, World!")
    for i := 0; i < 5; i++ {
        fmt.Printf("This is line %d\n", i)
        if i%2 == 0 {
            fmt.Println("Even number")
        } else {
            fmt.Println("Odd number")
        }
    }
    fmt.Println("End of printHello")
}

func printHelloTestB3() {
    fmt.Println("Hello, World!")
    for i := 0; i < 5; i++ {
        fmt.Printf("This is line %d\n", i)
        if i%2 == 0 {
            fmt.Println("Even number")
        } else {
            fmt.Println("Odd number")
        }
    }
}

func main() {
    printHelloTestB1()
    printHelloTestB2()
    printHelloTestB3()
}

File: tests/c/testB.c
#include <stdio.h>

void print_hello_test_b1() {
    printf("Hello, World!\n");
    for (int i = 0; i < 5; i++) {
        printf("This is line %d\n", i);
        if (i % 2 == 0) {
            printf("Even number\n");
        } else {
            printf("Odd number\n");
        }
    }
}

void print_hello_test_b2() {
    printf("Hello, World!\n");
    for (int i = 0; i < 5; i++) {
        printf("This is line %d\n", i);
        if (i % 2 == 0) {
            printf("Even number\n");
        } else {
            printf("Odd number\n");
        }
    }
}

int main() {
    print_hello_test_b1();
    print_hello_test_b2();
    return 0;
}